Dockerfile

```diff
@@ -100,9 +100,6 @@ RUN ./bin/run-db-download.py --force
 # get fresh l10n files
 RUN ./manage.py l10n_update
 
-# generate the sitemaps
-RUN ./manage.py update_sitemaps
```

Comment:
I removed this because the sitemaps take some time now that we're getting the translations and this step was only for the dev image where it's not really needed.